nntrainer/layers/common_properties.h

```diff
@@ -10,18 +10,29 @@
  * @author Jihoon Lee <jhoon.it.lee@samsung.com>
  * @bug    No known bugs except for NYI items
  */
-
-#include <string>
+#ifndef __COMMON_PROPERTIES_H__
+#define __COMMON_PROPERTIES_H__
 
 #include <array>
-#include <base_properties.h>
 #include <fstream>
+#include <string>
 
-#ifndef __COMMON_PROPERTIES_H__
-#define __COMMON_PROPERTIES_H__
+#include <base_properties.h>
 
 namespace nntrainer {
 
+/**
+ * @brief     Enumeration of activation function type
```

Comment:
add a note that if these enum are assigned values manually, update ActivationTypeInfo correspondingly,

Reply:
Updated